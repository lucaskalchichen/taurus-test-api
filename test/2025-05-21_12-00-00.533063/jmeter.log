2025-05-21 12:00:07,747 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 12:00:07,757 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 12:00:07,757 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 12:00:07,757 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063/jmeter-bzt.properties
2025-05-21 12:00:07,758 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063/system.properties
2025-05-21 12:00:07,761 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 12:00:07,762 INFO o.a.j.JMeter: Version 5.5
2025-05-21 12:00:07,762 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 12:00:07,762 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 12:00:07,762 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 12:00:07,762 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 12:00:07,762 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 12:00:07,763 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 12:00:07,763 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 12:00:07,763 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 12:00:07,763 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 12:00:07,765 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 12:00:07,766 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 12:00:07,766 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 12:00:07,766 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 12:00:07,766 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 12:00:07,772 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 12:00:07,772 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063
2025-05-21 12:00:07,772 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063
2025-05-21 12:00:07,773 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063
2025-05-21 12:00:07,779 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 12:00:07,780 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063'
2025-05-21 12:00:07,852 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 12:00:07,863 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 12:00:07,864 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 12:00:07,866 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063/modified_requests.jmx
2025-05-21 12:00:07,883 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 12:00:07,883 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 12:00:07,891 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:00:07,891 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:00:07,891 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:00:07,891 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:00:07,891 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 12:00:07,891 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 12:00:07,908 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 12:00:07,908 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 12:00:07,908 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 12:00:07,914 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 12:00:07,914 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 12:00:08,048 INFO o.a.j.JMeter: Running test (1747839608048)
2025-05-21 12:00:08,057 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 12:00:08,057 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 12:00:08,057 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 12:00:08,058 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 12:00:08,063 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 12:00:08,075 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 12:00:08,076 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 12:00:08,076 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 12:00:08,077 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 12:00:08,078 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 12:00:08,078 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 12:00:08,078 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 12:00:08,078 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 12:00:08,133 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 12:00:08,202 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 12:00:08,273 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 12:00:08,345 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 12:00:08,415 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 12:00:08,485 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 12:00:08,556 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 12:00:08,627 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 12:00:08,697 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 12:00:08,769 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 12:00:08,841 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 12:00:08,911 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 12:00:08,982 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 12:00:09,053 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 12:00:09,123 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 12:00:09,195 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 12:00:09,265 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 12:00:09,336 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 12:00:09,407 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 12:00:09,480 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 12:00:09,550 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 12:00:09,621 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 12:00:09,692 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 12:00:09,762 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 12:00:09,834 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 12:00:09,905 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 12:00:09,975 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 12:00:10,047 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 12:00:10,118 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 12:00:10,188 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 12:00:10,260 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 12:00:10,330 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 12:00:10,401 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 12:00:10,474 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 12:00:10,544 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 12:00:10,614 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 12:00:10,685 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 12:00:10,756 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 12:00:10,829 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 12:00:10,898 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 12:00:10,970 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 12:00:16,741 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 12:00:16,742 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 12:00:20,395 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 12:00:20,396 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 12:00:25,048 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 12:00:25,049 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 12:00:25,049 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 12:00:25,049 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 12:00:26,126 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 12:00:26,127 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 12:00:29,689 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 12:00:29,689 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 12:00:36,213 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 12:00:36,213 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 12:00:36,214 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 12:00:36,214 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 12:00:36,214 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 12:00:36,214 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 12:00:37,623 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 12:00:37,623 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 12:00:39,048 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 12:00:39,048 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 12:00:41,465 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 12:00:41,465 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 12:00:42,969 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 12:00:42,969 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 12:00:43,822 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 12:00:43,822 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 12:00:43,822 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 12:00:43,823 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 12:00:44,660 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 12:00:44,660 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 12:00:45,339 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 12:00:45,339 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 12:00:47,933 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 12:00:47,933 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 12:00:48,776 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 12:00:48,776 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 12:00:50,451 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 12:00:50,451 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 12:00:50,452 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 12:00:50,452 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 12:00:52,676 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 12:00:52,676 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 12:00:53,347 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 12:00:53,347 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 12:00:54,254 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 12:00:54,255 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 12:00:55,834 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 12:00:55,834 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 12:00:56,304 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 12:00:56,304 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 12:00:57,503 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 12:00:57,503 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 12:00:58,138 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 12:00:58,138 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 12:01:00,262 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 12:01:00,262 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 12:01:00,262 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 12:01:00,262 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 12:01:00,263 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 12:01:00,263 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 12:01:02,017 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 12:01:02,017 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 12:01:02,017 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 12:01:02,017 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 12:01:02,752 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 12:01:02,752 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 12:01:04,337 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 12:01:04,337 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 12:01:04,337 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 12:01:04,337 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 12:01:06,016 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 12:01:06,016 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 12:01:07,361 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 12:01:07,362 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 12:01:07,397 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 12:01:07,397 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 12:01:07,397 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 12:01:07,398 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 12:01:08,257 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 12:01:08,257 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 12:01:08,927 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 12:01:08,927 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 12:01:08,928 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 12:01:08,929 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
