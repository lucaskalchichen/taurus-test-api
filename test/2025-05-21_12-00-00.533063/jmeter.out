Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-00-00.533063/modified_requests.jmx
Starting standalone test @ 2025 May 21 12:00:07 ART (1747839607903)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 12:01:08 ART (1747839668929)
... end of run
